src/Header.js

```diff
@@ -1,11 +1,20 @@
 import React from "react";
+import { ToggleOn, ToggleOff, HeartPulse, HeartPulseFill } from 'react-bootstrap-icons';
 import './Header.css';
 
 class Header extends React.Component {
   render() {
+    console.log(this.props);
     return (
       <header>
         <h1>Horned Beast</h1>
+        <div id="headerInnerContainer">
+          {this.props.pulse ? <HeartPulseFill width="32" height="32" onClick={() => {this.props.onClick.handlePulseClick()}}/> :
+          <HeartPulse width="32" height="32" onClick={() => { this.props.onClick.handlePulseClick() }} />}
+          
+          {!this.props.darkTheme ? <ToggleOn width="40" height="40" onClick={() => {this.props.onClick.handleThemeClick()}} /> : 
+          <ToggleOff width="40" height="40" onClick={() => { this.props.onClick.handleThemeClick() }} />}
```

Comment:
Also very common:

3. Pass through a calculated prop, e.g.
    ```js
    <Toggle text={this.props.darkTheme ? 'Off' : 'On'} … />
    ```

Reply:
This also makes a lot of sense.  I wasn't sure what to do, since I am using BootStrap Icon components.  Your solution of dynamically changing the prop values I am passing to the BootStrap components seems much cleaner.  Thank you!

Reply:
@dahlbyk what are you referring to with the fill prop from <HeartPulse fill={this.props.pulse} … /> What is fill?

Reply:
`fill` would be a new prop that the `HeartPulse` component uses to behave like `HeartPulseFill` does now (thus the name). There may be a better name option, I just didn't go looking.